specification/azuredata/resource-manager/Microsoft.AzureData/preview/2019-07-24-preview/azuredata.json

```diff
@@ -1984,6 +2272,88 @@
         }
       }
     },
+    "SqlServerInstanceProperties": {
+      "description": "Properties of SqlServerInstance.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string",
+          "description": "null"
+        },
+        "edition": {
+          "type": "string",
+          "description": "null"
+        },
+        "containerResourceId": {
```

Comment:
>containerResourceId [](start = 9, length = 19)

what is a container in this context?

Reply:
It is resource id of Azure Arc for Servers. Only an on-premise machine is registered for Azure Arc for Servers can register its SQL Server instance as a sqlServerInstances resource.